opencog/atoms/execution/ExecutionOutputLink.cc

```diff
@@ -105,14 +101,19 @@ ExecutionOutputLink::ExecutionOutputLink(const Link& l)
 ///
 Handle ExecutionOutputLink::execute(AtomSpace* as, bool silent) const
 {
+	if (_outgoing[0]->getType() != GROUNDED_SCHEMA_NODE) {
+		LAZY_LOG_FINE << "Not a grounded schema. Do not execute it";
```

Comment:
Oh right. OK.  In some future, it would make more sense to do "lazy evaluation" and unfold, only if the execution link was actually hit. But that's for later.